src/main/java/org/nibor/autolink/LinkExtractor.java

```diff
@@ -39,6 +41,22 @@ public static Builder builder() {
             }
         };
     }
+    /**
+     * Extract all links form input text to list of strings.
+     * @param input - the input text
+     * @return the links from input text as list of strings
+     * @throws NullPointerException if the input argument is null
+     * @see {@link String}, {@link List}
+     * 
+     */
+    public List<String> extractLinksToListOfStrings(CharSequence input) {
```

Comment:
I think we can shorten this method's name to `extractLinkTexts` or something similar.